packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/index.scss

```diff
@@ -32,7 +32,7 @@
 
     &__container {
         @include desktop {
-            height: calc(80vh - 0.8rem - 4rem - 4.8rem - 5.8rem);
+            height: calc(80vh - 0.8rem - 4rem - 4.8rem - 13rem);
 
             &--has-footer {
                 height: calc(80vh - 0.8rem - 4rem - 4.8rem - 5.8rem - 7.4rem);
```

Comment:
Can we finalize the amount to deduct on `80vh` and leave a comment of the breakdown instead?